demo/build.gradle

```diff
@@ -14,6 +14,8 @@ android {
 }
 
 dependencies {
-    compile project(":core")
+    compile (project(":core")) {
+        exclude group: 'com.android.support', module: 'support-core-ui'
```

Comment:
this also showcases now how to use a different version of the support library :)